### PR TITLE
List Nav and Modal

### PR DESCRIPTION
**Pull request by iamharnad**
Added List, navigation and Modal components

### CODE REVIEW

File: Components/modal.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.modal-interstitial {
+  display: none;
+  z-index: var(--z-index-6);
+  padding: 0 1rem;
+}
+.modal-content {
+  background-color: rgb(252, 240, 240);
+  position: relative;
+  max-width: 600px;
```
Comment (amarphule) [inline Components/modal.css:9]: use `rem`

File: Components/modal.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.modal-interstitial {
+  display: none;
+  z-index: var(--z-index-6);
+  padding: 0 1rem;
+}
+.modal-content {
+  background-color: rgb(252, 240, 240);
```
Comment (amarphule) [inline Components/modal.css:7]: use color variable

File: Components/modal.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  right: 7px;
+}
+
+.active {
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background-color: rgba(0, 0, 0, 0.5);
```
Comment (amarphule) [inline Components/modal.css:27]: same here also use color variable name

File: documentation.html [hunk 4]
```diff
@@ -102,6 +103,9 @@ <h2 class="h5 nav-title">Components</h2>
           <li class="nav-link-item"><a href="#Form">Form</a></li>
           <li class="nav-link-item"><a href="#Image">Image</a></li>
           <li class="nav-link-item"><a href="#Input">Input</a></li>
+          <li class="nav-link-item"><a href="#List">List</a></li>
```
Comment (amarphule) [inline documentation.html:104]: The typo you used `List` instead of `Lists`. that's why a link is not working

File: documentation.html [hunk 5]
```diff
@@ -768,6 +772,277 @@ <h6 class="text-regular-weight product-title">
...
+              class="styled-list flex-items list-container"
+              style="background-color: #d3d3d3"
+            >
+              <li>list item1</li>
+              <li>list item2</li>
+              <li>list item3</li>
+            </ul>
+            <ul
+              class="styled-list list-style-square flex-items list-container"
+              style="background-color: #d3d3d3"
```
Comment (amarphule) [inline documentation.html:807]: don't use Inline style

**Review by amarphule — CHANGES_REQUESTED** [review]
Well done @iamharnad , Your site looks good and the code is well structured.
I do have some suggestions for you
1. don't use inline-style
2. use a variable name for a color
3. check the contrast ratio

**Review by HrishikeshMahalle — APPROVED** [review]
Please have a look at all the changes mentioned by Amar phule. Rest all looks Gr8. Awsm Job. Solve the conflicts as well, they are Hard to address so do Have an eye on them.

**Pull request merged**